src/playback/summary.jsx

```diff
@@ -140,30 +140,36 @@ class PlaybackSummary extends React.Component {
         score:
           lookupType[
             selectFrom(speeds)
-              .map(({ speed, loss }) => {
-                if (speed === 1) {
-                  if (missing(loss)) return 3;
-                  if (loss > 1) return 2;
-
-                  return 0;
-                }
-
-                if (missing(loss) || loss <= 1) return 0;
-
-                return 1;
-              })
+              .map(({ size, speed, loss }) => this.getPlaybackScore(
+                size, speed, loss, sizes.slice(0, 3),
```

Comment:
`sizes.slice(0, 3),` is magic.  Please update the config so that you do not need special code like this.